### PR TITLE
Add proto3 dependency

### PR DESCRIPTION
**Pull request by kaikawaliu**
## Backward Compatibility

- [x] This is not a breaking change
- [ ] This is a breaking change

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

~- [ ] local machine test~

- [x] small-scale cloud test

### CODE REVIEW

File: src/libPersistence/ScillaMessage.proto [hunk 1]
```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+message ProtoScillaVal {
+	message Map {
+		map<string, ProtoScillaVal> m = 1;
```
Comment (kaikawaliu) [inline src/libPersistence/ScillaMessage.proto]: bytes cannot be used as key in proto map, only string can.

**Comment by Gnnng** [comment]
Several pieces of information seem to be missing:
* Install instructions: How a user can install the new proto3? (users cannot directly run `scripts/download_proto3.sh` because some steps there requires root permissions but running the entire script in root is not safe)
* Compatibility clarification: Will installing proto3 be conflicting with existing proto2 on the user machine?

Also, please kindly check all the places where the apt packages need to be updated.

**Comment by ansnunez** [comment]
> Several pieces of information seem to be missing:
> 
>     * Install instructions: How a user can install the new proto3? (users cannot directly run `scripts/download_proto3.sh` because some steps there requires root permissions but running the entire script in root is not safe)
> 
>     * Compatibility clarification: Will installing proto3 be conflicting with existing proto2 on the user machine?
> 
> 
> Also, please kindly check all the places where the apt packages need to be updated.

@Gnnng for Install instructions, will it suffice to add the call to our README after the other root-level commands? https://github.com/Zilliqa/Zilliqa#build-dependencies

**Comment by ansnunez** [comment]
Fixed in #1778

**Pull request closed without merging**